### PR TITLE
improve readmes

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
Just some details I found while going through the documentation.

### CODE REVIEW

File: digital-ocean/container-linux/kubernetes/README.md [hunk 1]
```diff
@@ -1,23 +1 @@
...
-* Single or multi-master, [flannel](https://github.com/coreos/flannel) networking
-* On-cluster etcd with TLS, [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)-enabled
-* Advanced features like [snippets](https://typhoon.psdn.io/advanced/customization/#container-linux) customization
-* Ready for Ingress, Prometheus, Grafana, CSI, and other [addons](https://typhoon.psdn.io/addons/overview/)
-
-## Docs
-
-Please see the [official docs](https://typhoon.psdn.io) and the Digital Ocean [tutorial](https://typhoon.psdn.io/cl/digital-ocean/).
-
+Support for Digital Ocean isn't under active developing now, so there is not an updated documentation for it.
```
Comment (invidian) [inline digital-ocean/container-linux/kubernetes/README.md]:
```suggestion
Support for Digital Ocean isn't under active developing now, so there is no up-to-date documentation for it.
```

File: docs/flatcar-linux/kvm-libvirt.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ A generated `kubeconfig` provides `kubectl` access to the cluster.
 
 ## Prepare VM Image
 
-Download the Flatcar image. The example below uses the "Edge" Flatcar channel.
+Download the Flatcar Linux image. The example below uses the "Edge" Flatcar Linux channel.
```
Comment (invidian) [inline docs/flatcar-linux/kvm-libvirt.md]:
```suggestion
Download the Flatcar Container Linux image. The example below uses the "Edge" Flatcar Container Linux channel.
```

File: docs/flatcar-linux/kvm-libvirt.md [hunk 3]
```diff
@@ -319,7 +319,7 @@ source.
 | asset_dir | Path to a directory where generated assets should be placed (contains secrets) | "/home/user/infra/assets" |
 | cluster_name | Unique cluster name | "vmcluster" |
 | machine_domain | DNS zone | "vmcluster.k8s" |
-| os_image_unpacked | Path to unpacked Flatcar image (probably after a qemu-img resize IMG +5G) | "file:///home/user/infra/flatcar_production_qemu_image.img" |
+| os_image_unpacked | Path to unpacked Flatcar Linux image (probably after a qemu-img resize IMG +5G) | "file:///home/user/infra/flatcar_production_qemu_image.img" |
```
Comment (invidian) [inline docs/flatcar-linux/kvm-libvirt.md]:
```suggestion
| os_image_unpacked | Path to unpacked Flatcar Container Linux image (probably after a qemu-img resize IMG +5G) | "file:///home/user/infra/flatcar_production_qemu_image.img" |
```

File: docs/flatcar-linux/kvm-libvirt.md [hunk 4]
```diff
@@ -350,7 +350,7 @@ source.
 | libvirtbaseid | The cluster's OS volume to use for copy-on-write | "${module.controller.libvirtbaseid}" |
 | libvirtpool | The cluster's disk volume pool to use | "${module.controller.libvirtpool}" |
 | machine_domain | DNS zone | "${module.controller.machine_domain}" |
-| os_image_unpacked | Path to unpacked Flatcar image (probably after a qemu-img resize IMG +5G) | "file:///home/user/infra/flatcar_production_qemu_image.img" |
+| os_image_unpacked | Path to unpacked Flatcar Linux image (probably after a qemu-img resize IMG +5G) | "file:///home/user/infra/flatcar_production_qemu_image.img" |
```
Comment (invidian) [inline docs/flatcar-linux/kvm-libvirt.md]:
```suggestion
| os_image_unpacked | Path to unpacked Flatcar Container Linux image (probably after a qemu-img resize IMG +5G) | "file:///home/user/infra/flatcar_production_qemu_image.img" |
```

**Review by invidian — CHANGES_REQUESTED** [review]
Suggested some nits, nice findings @mauriciovasquezbernal :+1:

File: docs/flatcar-linux/kvm-libvirt.md (hunk 2 above)
Comment (mauriciovasquezbernal) [reply docs/flatcar-linux/kvm-libvirt.md]: I got it almost right :rofl:

**Comment by mauriciovasquezbernal** [comment]
@invidian I committed your suggested changes, I'll force push in a single commit in few seconds. Thanks for the review.

**Review by invidian — APPROVED** [review]

**Pull request merged**